manifests/init.pp

```diff
@@ -66,4 +69,32 @@
       }
     }
   }
+
+  if $gsutil_install {
+    # TODO: Windows support.
+    if $facts['os']['family'] != 'Windows' {
+      # Using bundled install option:
+      # https://cloud.google.com/storage/docs/quickstart-gsutil
+
+      file { '/opt/gsutil-bundle':
+        ensure => 'directory',
+      }
+
+      archive { 'gsutil.zip':
```

Comment:
I was really hoping to find packages for linux distros that we could manage instead, but I failed :(

Reply:
it's Google.